app/views/default/_proposal_properties.html.erb

```diff
@@ -10,7 +10,11 @@
         <strong><%= name %>
       </td>
       <td>
-        <%= value %>
+        <% if value.class == BigDecimal %>
+          <%= number_to_currency(value) %>
```

Comment:
Hmm, I wonder if we can safely assume all decimals should be displayed as $$...ok for now I suppose.